<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericTab</class>
 <widget class="QWidget" name="GenericTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="toolbarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="mainToolBarLayout">
      <item>
       <widget class="QPushButton" name="loadPushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Load file</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../gui_res.qrc">
          <normaloff>:/Images/icons/document-open-8.png</normaloff>:/Images/icons/document-open-8.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="historyUndoPushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Undo</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../gui_res.qrc">
          <normaloff>:/Images/icons/edit-undo-8.png</normaloff>:/Images/icons/edit-undo-8.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+Z</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="historyRedoPushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Redo</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../gui_res.qrc">
          <normaloff>:/Images/icons/edit-redo-8.png</normaloff>:/Images/icons/edit-redo-8.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+Shift+Z</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="toolLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gui_res.qrc"/>
 </resources>
 <connections/>
</ui>
